community/shellinabox/APKBUILD

```diff
@@ -0,0 +1,61 @@
+# Contributor: Ishak BELAHMAR <ishak@nytrio.com>
+# Maintainer: Ishak BELAHMAR <ishak@nytrio.com>
+pkgname=shellinabox
+pkgver=3.9.6
+pkgrel=0
+pkgdesc="publish command line shell through AJAX interface"
+url="https://github.com/shellinabox/shellinabox"
+arch="x86_64"
+license="GPL-2.0"
+depends="zlib musl"
+makedepends="libressl-dev linux-pam-dev zlib-dev autoconf automake libtool g++"
+install="$pkgname.post-install"
+subpackages="$pkgname-doc"
+source="${pkgname}-${pkgver}.tar.gz::https://github.com/shellinabox/shellinabox/archive/master.tar.gz launcher.patch"
+builddir="$srcdir"/shellinabox-master
+
+
+prepare() {
+	patch -p0 -i "$srcdir"/launcher.patch
+}
+
+build() {
+	cd "$builddir"
+	autoreconf -i
+	./configure
+	make
+}
+
+check() {
+	cd "$builddir"
+}
```

Comment:
The check function can be omitted. Add before depends, `options="!check" # No test suite available`